packages/backend-common/src/reading/FetchUrlReader.ts

```diff
@@ -39,18 +40,28 @@ export class FetchUrlReader implements UrlReader {
    * `host`:
    *   Either full hostnames to match, or subdomain wildcard matchers with a leading `*`.
    *   For example `example.com` and `*.example.com` are valid values, `prod.*.example.com` is not.
+   *
+   * `paths`:
+   *   An optional list of paths which are allowed. If the list is omitted all paths are allowed.
    */
   static factory: ReaderFactory = ({ config }) => {
     const predicates =
       config
         .getOptionalConfigArray('backend.reading.allow')
         ?.map(allowConfig => {
+          const paths = allowConfig.getOptionalStringArray('paths');
+          const checkPath = paths
+            ? (url: URL) => {
+                const targetPath = normalizePath(url.pathname);
```

Comment:
Not sure about the use of `normalize` here. Firstly: what does it achieve in this particular context? Secondly: feels like this might break on Windows, feels like you would need to do `path.posix.normalize` here to guarantee that it expects forward slashes.

If I'm not mistaken, it feels like it would be better to skip the normalization and instead do

```ts
(url: URL) => paths.some(path => url.pathname === path || url.pathname.startsWith(`@{path}/`))
```